contracts/GenArt721CoreV3.sol

```diff
@@ -744,6 +799,7 @@ contract GenArt721CoreV3 is
         external
         onlyUnlocked(_projectId)
         onlyArtistOrAdminACL(_projectId, this.updateProjectName.selector)
+        onlyNonEmptyString(_projectName)
```

Comment:
nit: should we also be using `onlyValidProjectId ` since adminACL can call this?

Reply:
See: https://github.com/ArtBlocks/artblocks-contracts/pull/310#discussion_r981381481